src/app/shared/services/dia-backend/series/dia-backend-series-repository.service.ts

```diff
@@ -0,0 +1,52 @@
+// tslint:disable: no-magic-numbers
+import { HttpClient, HttpErrorResponse } from '@angular/common/http';
+import { Injectable } from '@angular/core';
+import { defer, Subject, throwError } from 'rxjs';
+import { catchError, concatMap } from 'rxjs/operators';
+import { DiaBackendAuthService } from '../auth/dia-backend-auth.service';
+import { NotFoundErrorResponse } from '../errors';
+import { BASE_URL } from '../secret';
+
+@Injectable({
+  providedIn: 'root',
+})
+export class DiaBackendSeriesRepository {
+  private readonly postCapturesUpdated$ = new Subject<{ reason?: string }>();
+
+  constructor(
+    private readonly httpClient: HttpClient,
+    private readonly authService: DiaBackendAuthService
+  ) {}
+
+  readSeries$() {
+    return defer(() => this.authService.getAuthHeaders()).pipe(
+      concatMap(headers =>
+        this.httpClient.get<DiaBackendSeries>(`${BASE_URL}/api/v2/series/`, {
+          headers,
+        })
+      ),
+      catchError((err: unknown) => {
+        // eslint-disable-next-line @typescript-eslint/no-magic-numbers
+        if (err instanceof HttpErrorResponse && err.status === 404) {
+          return throwError(new NotFoundErrorResponse(err));
+        }
+        return throwError(err);
+      })
+    );
+  }
+}
+
+export interface DiaBackendSeries {
+  readonly id: string;
+  readonly collections: string;
+  readonly in_store: boolean | null | undefined;
```

Comment:
I believe the API will not return `undefined`. Thus, use `boolean | null` only.
```suggestion
  readonly in_store: boolean | null;
```